lib/medium-admin.php

```diff
@@ -764,13 +765,26 @@ public static function cross_post($post, $medium_post, $medium_user) {
       $path = "/v1/users/{$medium_user->id}/posts";
     }
 
-
     return self::_medium_request("POST", $path, $medium_user->token, $data, array(
       "Content-Type" => "application/json"
     ));
   }
 
   /**
+   * Creates a post claim for the original author of a post.
+   */
+  public static function create_post_claim(Medium_Post $medium_post, Medium_User $medium_user, $user) {
+    $body = array(
+      "md5" => md5($user->user_email)
```

Comment:
Good idea. Not technically to spec, but will probably save issues.
On Wed, Dec 2, 2015 at 9:51 PM Mikko T. <notifications@github.com> wrote:

> In lib/medium-admin.php
> <https://github.com/Medium/medium-wordpress-plugin/pull/50#discussion_r46514907>
> :
>
> >      return self::_medium_request("POST", $path, $medium_user->token, $data, array(
> >        "Content-Type" => "application/json"
> >      ));
> >    }
> >
> >    /**
> > +   * Creates a post claim for the original author of a post.
> > +   */
> > +  public static function create_post_claim(Medium_Post $medium_post, Medium_User $medium_user, $user) {
> > +    $body = array(
> > +      "md5" => md5($user->user_email)
>
> We should probably downcase the email to make the comparison
> case-insensitive across WP/Medium
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Medium/medium-wordpress-plugin/pull/50/files#r46514907>
> .
>